src/synthesized_insight/metrics/base.py

```diff
@@ -83,6 +84,81 @@ def __call__(self, sr_a: pd.Series, sr_b: pd.Series) -> Tuple[Union[int, float,
         return self._compute_test(sr_a, sr_b)
 
 
+class TwoColumnMap(_Metric):
```

Comment:
Could we also move these classes to a new file? They're not really base classes

Reply:
Moved these to new fil as suggested